gatsby-config.js

```diff
@@ -438,6 +438,39 @@ module.exports = {
         nodesPerFeedFile: Infinity,
       },
     },
+    {
+      resolve: `gatsby-plugin-json-output`,
```

Comment:
Out of curiosity, why use the `json-output` plugin here instead of our `generate-json` plugin?

Reply:
The previous script expects pagination, which we get for free from the `json-output` plugin. It looks like our plugin creates a single JSON file, which would be far too big for _all_ the release notes.

Reply:
Got it. How are we planning to handle pagination? Because we can't build pages with query params, are we going to do some redirect magic in Amplify? Or modify the service that handles this?

Reply:
This creates `release-notes-1.json`, `release-notes-2.json`, etc and python script handles crawling the pages. Ideally we would upload these files directly to s3 and remove the need for a python script to fetch these, but we don't currently have an IAM role with access to the necessary bucket.

Reply:
> This creates release-notes-1.json, release-notes-2.json

Totally understand that. Sorry, I should have been more clear what I'm asking. Let me try again 😄 

> Ideally we would upload these files directly to s3

This is really more of what I'm poking at. Is this "ideal" scenario going to happen right away?

If we release this change now, will the python script get updated to work with the new format, or will it continue to expect to paginate between pages using query params? In my mind, one of 3 things need to happen in order to get this working again.

1. Get the solution in place to upload this data directly to s3
2. Get the python script updated to use the new endpoint format `-1.json`, `-2.json` for pagination
3. Add redirect magic to amplify to rewrite `?page=2` requests to `-2.json` endpoints

Without any of these changes, we won't be able to source the data correctly. I was more curious on what we plan to do **now** once this change is merged.

Reply:
My aim was to take the fastest, least disruptive, approach to solving the problem. Since we already had a python script in place, we had an engineer willing to update it, and we had a plugin set up to created paginated results in this format, I thought this was the best direction.

Since this PR isn't building at the moment, and this is turning out to _not_ be a quick fix, I'll start looking into what it would take to set this up with S3.

Reply:
After speaking with Erika and Vince, we've decided to just supply one JSON file and have NerdGraph reach out to us directly. I've updated the code and the PR description to reflect this.